templates/index.html

```diff
@@ -37,8 +37,7 @@ <h2 class="feature-title">Data Driven</h2>
             All engine and game logic uses Bevy ECS, a custom Entity Component System
             <ul class="feature-sublist">
               <li>
-                <b>Fast:</b> Massively Parallel and Cache-Friendly. The fastest ECS according to some
-                benchmarks
+                <b>Fast:</b> Massively Parallel and Cache-Friendly.
```

Comment:
Can be addressed in a different PR, but what remains here might also need some rewording. Cache-friendly might be a bit niche for those not familiar with data oriented programming.